File: CPP/Assignment/day2/factorial.cpp
#include<iostream>
using namespace std;

double calculateFactorial(int n){
	double prod=1;
	for(int i=n;i>1;i--){
	  prod = prod*i;
	}
	return prod;

}

int main(){
	int n;

	cout<<"Enter value : "<<endl;
	cin>>n;
	cout<<"Factorial Series : ";
	for(int i=1; i<=n ; i++)
	{
	  cout<<calculateFactorial(i)<<" ";
	}
	cout<<endl;
	return 0;
}

File: OOPWJAVA/Assignment/Day1/src/Factorial.java
/* 5) Write a function for factorial and use the function for printing factorial series like 1 2 6 24 120……… */

import java.util.Scanner;
public class Factorial{
	
	
	public static int factorial(int num){
		int fact=1;
		for(int i=1;i<=num;i++)
		{
			fact=fact*i;
		}
		return fact;
		
		
	}
	
	public static void main(String[] args){
		int num;
		System.out.println("Enter term: ");
		Scanner sc= new Scanner(System.in);
		num = sc.nextInt();
		for(int i=1;i<=num;i++)
	{
		System.out.print(factorial(i) + " ");	
	}	
	}	
}